reference-test/src/test/java/au/org/consumerdatastandards/conformance/APIStepsBase.java

```diff
@@ -29,6 +34,27 @@ void setupApiBasePath(String apiBasePath) {
         this.apiBasePath = apiBasePath;
     }
 
+    @Step("Setup PTT properties")
+    void setupPttProperties() throws IOException {
+        props = new Properties();
+        props.load(APIStepsBase.class.getResourceAsStream("/ptt.properties"));
+
+        SSLConfig sslConfig = SSLConfig.sslConfig().allowAllHostnames();
```

Comment:
@fyang1024 `allowAllHostnames()` replaces `relaxedHTTPSValidation()` here. It's basically similar, but works with MTLS unlike `relaxedHTTPSValidation()`.